modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_buttons.scss

```diff
@@ -17,4 +17,14 @@
     @include modular-spacing('margin-bottom', 0);
   }
 
+  // If they're in the same paragraph, get them to stack.
+  a[class^="su-button"] {
```

Comment:
You really don't need the attribute selector for this. You can use `a.su-button` or with sass 
```
a {
  &.su-button {
```

Reply:
I am using this to select: 

- su-button
- su-button--big
- su-button--secondary

When I use 
```
a {
  &.su-button {

```
I doesn't match.

I thought there was a better way in SASS using `*` but I can't find it.